kong/runloop/handler.lua

```diff
@@ -707,6 +708,17 @@ return {
             header[upstream_status_header] = matches[0]
           end
         end
+
+        if ctx.SET_HASH_ON_COOKIE then
```

Comment:
I put this in the `before` hook, presumably this will give plugins the ability to access/ modify the cookie value if there is some use case for it.